src/she/plaintext.rs

```diff
@@ -28,6 +28,16 @@ impl Plaintexts {
         Plaintexts { vals: res }
     }
 
+    pub fn restricted_rand<T: Rng>(params: &SHEParameters, rng: &mut T) -> Plaintexts {
+        let upper_bound = 1000000000;
+        let lower_bound = 100000;
```

Comment:
How did you decided this upper bound & lower bound?

Reply:
need to satisfy the following approximate conditions.
Lower Bound > maximum value of possible secret input (approximately 10,000 in this case) * number of participants
Upper Bound * number of participants < period of the ScalarField of edwards_bls12_377 (≒10^75)

Reply:
Got it! I added comment